### PR TITLE
Properly disable ASM on targets where we can't use SIMD instructions

### PR DESCRIPTION
**Pull request by NiLuJe**
Hopefully unfucks the situation on Android...

### CODE REVIEW

File: Makefile.defs [hunk 1]
```diff
@@ -463,7 +463,8 @@ ifneq (,$(findstring march=arm,$(CFLAGS)))
 	# We're on ARM
 	ifeq (,$(findstring fpu=neon,$(CFLAGS)))
 		# We don't support NEON
-		undefine WANT_SIMD
+		# NOTE: undefine is new to GNU make 3.8...
```
Comment (Frenzie) [inline Makefile.defs]: Umm where do we come across this make from, what, 2010? Older? >_>

Comment (Frenzie) [reply Makefile.defs]: Oh, the Android TC? Makes sense. They also use some shell scripting methods that were deprecated around the year I was born, so… :-P

Comment (NiLuJe) [reply Makefile.defs]: Android, apparently ;D

Comment (NiLuJe) [reply Makefile.defs]: That's should say gmake 3.82, but, still ;p.

**Pull request merged**